project/Dependencies.scala

```diff
@@ -37,7 +37,8 @@ object Dependencies {
   val cats: ModuleID =           "org.typelevel"              %% "cats"            % "0.9.0"
   val enumeratum: ModuleID =     "com.beachape"               %% "enumeratum"      % "1.5.13"
 
-  val javaxServlet = "javax.servlet" % "servlet-api" % "2.5" % "provided"
+  val javaxServlet = "javax.servlet" % "javax.servlet-api" % "3.1.0" % "provided"
+  val javaxWsRS = "javax.ws.rs" % "javax.ws.rs-api" % "2.0"
```

Comment:
Are we using `javax.ws.rs` (JAX-RS) currently? If not, maybe remove it from `Dependencies` until we decide to use it.